libs/api/domains/official-journal-of-iceland/src/lib/models/advert.input.ts

```diff
@@ -61,6 +64,9 @@ export class QueryParams {
 
   @Field(() => Number, { nullable: true })
   page?: number
+
+  @Field(() => Number, { nullable: true })
```

Comment:
Perhaps we'd like to mark the GraphQl type as being an Int instead of using Number. https://graphql.org/graphql-js/basic-types/ 

However marking it as a Number still works though

Reply:
Thanks for the info!